enterprise/server/self-hosting-hunspell.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Add Hunspell dictionaries to Spell Checker Pro
+title_nav: Using Hunspell dictionaries
+description: Instructions for adding Hunspell dictionaries to TinyMCE Spell Checker Pro.
+keywords: enterprise tinymcespellchecker spell check checker pro server configuration configure
+---
+
+{{site.requires_5_9v}}
+
+{{site.productname}} 5.9 introduced support for Hunspell dictionaries. Hunspell dictionaries can be obtained from various sources, but the files must be stored to a strict structure to work with {{site.productname}} Spell Checker Pro. {{site.companyname}} provides dowloadable bundles that have the required structure.
```

Comment:
> stored to a strict structure

That's a strange wording.

Reply:
> ...must be stored in a specific structure to work...

Reply:
fixed in [`c367b42` (#2048)](https://github.com/tinymce/tinymce-docs/pull/2048/commits/c367b42164d11a3902f0b6d6cc4e0226b58a2d96)